Added Vacation-Ready content

cecilysullivan opened the pull request:
work in progress
@cwarnermm will help me put in embedded video with Ian discussing vacation-readiness, link for OOO email template, and screenshots of auto-declining in google calendar and MM.

cecilysullivan committed:
Added Vacation-Ready content

lynn915 reviewed (commented):
From the proposed text, remove "Here at Mattermost.....while they are out" 
Change the location of where this is being added and instead, put it under "Communicating Time Off". Update point 6 with the text. 
- Be Vacation-Ready: Assign PTO Backups; List your PTO Backups in GSuite Vacation Responder (insert link to example); Set your GCal to OOO and auto-decline new and existing meetings; change your Mattermost status and set automatic direct message replies (include existing link).
Add embedded video of Ian's COM overview of being vacation ready.

cecilysullivan committed:
Added some docs links (still need two more)

cwarnermm committed:
Added new OOO example page

cwarnermm committed:
Updated vacation-ready & added video link

A video has been added as a link that will take users with appropriate access to the Mattermost Community Server where the video is hosted.
To embed a video on this page, the video must be publicly available, and can't reside on a server where access is limited. Attempting to embed a video that isn't publicly available will result in no users being able to view the embedded video directly from the Handbook page.

cwarnermm committed:
Added OOO email example page to navigation pane

lynn915 reviewed (changes requested):
Everything looks great but can we remove the OOO template text and CEO example?   I created a document with this content for the onboarding slide deck, let's link that in two places: 
1. In the bullet "Assign PTO backups and list your PTO backups in GSuite Vacation Responder". Replace the link that is currently there that links to the handbook with this doc: https://docs.google.com/document/d/1Yu4pNey7BSjSG7eqEb6K4NDpmudOvGD7vbcdFyZegDg/edit?usp=sharing
2. In the bullet "when writing an out of office email response, the following recommendations help guide your coworkers on who they can contact in your absence". Add the same doc link to "out of office email response" : https://docs.google.com/document/d/1Yu4pNey7BSjSG7eqEb6K4NDpmudOvGD7vbcdFyZegDg/edit?usp=sharing

cwarnermm committed:
Updated OOO example link

cwarnermm committed:
Removed duplicate content from OOO example page
